src/middleware/validateApiKey.js

```diff
@@ -64,28 +24,16 @@ export default async (req, res, next) => {
     if (!apiKey && isDevelopment) {
       apiKey = FALLBACK_API_KEY;
     }
-    if (!apiKey) {
-      throw new Error('X-API-Key Header doesn\'t exist');
-    }
```

Comment:
it doesn't exist in your project version because you deleted that line - but if you take a look at this file but on the main branch in GitHub, you will see that the following is still in this file:

```js
if (!apiKey) {
  throw new Error('X-API-Key Header doesn\'t exist');
}
```

you don't want to delete this line because we want to catch the case where no API token is provided and we throw an error. if we don't, then we could allow the application to throw an even more obscure and detrimental issue down the line

so my recommendation is that you make as little changes to this file as possible to avoid introducing new possible bugs